TEAMS_EXTENSION.md

```diff
@@ -0,0 +1,126 @@
+|Azure Search Index (p) | The name of your Azure Cognitive Search Index. |
+|Azure Search Key (p) | An admin key for your Azure Cognitive Search resource. |
+|Azure OpenAI resource (p) | The name of your Azure OpenAI resource. This resource must have already been created previously. |
+|Azure OpenAI key (p) | The access key is associated with your Azure OpenAI resource. |
+|Orchestration strategy (p) | Use Azure OpenAI Functions (openai_functions) or LangChain (langchain) for messages orchestration. If you are using a new model version 0613 select "openai_functions" (or "langchain"), if you are using a model version 0314 select "langchain". |
+|Azure Form Recognizer Endpoint (p) | The name of the Azure Form Recognizer for extracting the text from the documents. |
+|Azure Form Recognizer Key (p) | The key of the Azure Form Recognizer for extracting the text from the documents. |
+|Azure Blob Account Name (p) | The name of the Azure Blob Storage for storing the original documents to be processed. |
+|Azure Blob Account Key (p) | The key of the Azure Blob Storage for storing the original documents to be processed. |
+|Azure Blob Container Name (p) | The name of the Container in the Azure Blob Storage for storing the original documents to be processed. |
+
+You can find the [ARM template](/extensions/infrastructure/main.json) used, along with a [Bicep file](/extensions/infrastructure/main.bicep) for deploying this accelerator in the /infrastructure directory.
+
+### Deploy Teams application
+1. Clone this GitHub repo.
+1. Open the “extensions/teams” folder with Visual Studio Code 
+
+![Teams](/media/teams.png) 
+1. Open the file env\.env.test
+
+![ENV](/media/teams-1.png) 
+1. Locate the environment variable AZURE_FUNCTION_URL.
+1. Replace the <YOUR AZURE FUNCTION NAME> with the name of your Function App resource (created in previous section)
+1. Save the file.
+1. Select Teams Toolkit from the navigation panel. 
+
+![Teams Toolkit in VS Code](/media/teams-2.png) 
+1. Within the Teams Toolkit panel, login to the following accounts:
+  **Sign in to Microsoft 365**: Use your Microsoft 365 work or school account with a valid E5 subscription for building your app. If you don't have a valid account, you can join [Microsoft 365 developer program](https://developer.microsoft.com/microsoft-365/dev-program) to get a free account before you start.
+  **Sign in to Azure**: Use your Azure account for deploying your app on Azure. You can [create a free Azure account](https://azure.microsoft.com/free/) before you start.
+
+![Teams Toolkit Accounts](/media/teams-3.png)
+2. Under **Environment**, select **test**.
+
+![Teams Toolkit Environment](/media/teams-4.png)
+3. Under **Lifecycle**, select **Provision**.
+
+![Teams Toolkit Lifecycle Provision](/media/teams-5.png)
+4. Within the command palette, select **test** for the environment.
+
+![Select an Environment](/media/teams-6.png) 
+5. Select the resource group created earlier in the installation
```

Comment:
Needs a new line after each image in this numbered list as text is not aligned

Reply:
Good find. Done.